cmd/rookd/main.go

```diff
@@ -57,12 +58,14 @@ type config struct {
 	publicIPv4         string
 	privateIPv4        string
 	devices            string
+	metadataDevice     string
```

Comment:
it could, but i prefer it being alongside the desired data devices.  it's really just another type of device the user is giving us permission to use.